sdk/python/feast/infra/feature_servers/multicloud/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mysqlclient
+feast[aws,gcp,snowflake,redis,go]
```

Comment:
you should be able to use feast[mysql] as well!

Reply:
done